proposed/phpdoc.md

```diff
@@ -173,27 +173,27 @@ interpreted as described in [RFC 2119][RFC2119].
 
   The following notations can be used per type of "Structural Element":
 
-  *Namespace*:      `\My\Space`
-  *Function*:       `\My\Space\myFunction()`
-  *Constant*:       `\My\Space\MY_CONSTANT`
-  *Class*:          `\My\Space\MyClass`
-  *Interface*:      `\My\Space\MyInterface`
-  *Trait*:          `\My\Space\MyTrait`
-  *Method*:         `\My\Space\MyClass::myMethod()`
-  *Property*:       `\My\Space\MyClass::$my_property`
-  *Class Constant*: `\My\Space\MyClass::MY_CONSTANT`
+  * *Namespace*:      `\My\Space`
```

Comment:
Are the leading asterisks here necessary?  I realize they're trying to be bullets, but I've learned to avoid using asterisks and underscores in Markdown unless I want italics, because that's typically how they've always been parsed.  Since no bullets were required beforehand, maybe they can continue to be left off.

Reply:
Yes, but that part is indented so renders in a pre-formatted block

Reply:
Ah, ok... if the bullets are rendering good for you in this top section, then maybe that's fine to keep.

Reply:
I could switch to `-` for bullets if you'd prefer, to avoid confusion with using `*` for that and italics?